.github/workflows/website.yml

```diff
@@ -97,26 +95,44 @@ jobs:
           web3_token: ${{ secrets.WEB3_TOKEN }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update IPFS DNSLink https://staging.web3.storage
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        run: npx dnslink-cloudflare --record staging --domain web3.storage --link /ipfs/${{ needs.build.outputs.cid }}
```

Comment:
shouldn't this use CID from "add to web3.storage", what is `needs.build.outputs.cid` ?